tensorflow/compiler/xla/stream_executor/cuda/BUILD

```diff
@@ -615,7 +616,11 @@ cc_library(
         ":cudnn_plugin",
         ":cufft_plugin",
         ":curand_plugin",
-    ],
+        ":cusolver_lib",
+        ":cusparse_lib",
+    ] + if_tensorrt([
+        "//tensorflow/compiler/tf2tensorrt:tensorrt_lib",
```

Comment:
Why are we adding the `_lib` targets to the all_runtime here? Should they be dependencies of the targets that actually need them?

Reply:
IIRC, the linkopts with the RPATH that we need from these targets doesn’t get carried through to libtensorflow unless we explicitly list them out. I’m not sure if that is because there is actually no dependencies from the sub targets that need them or because of some quirk of bazel.

We could instead make a new target which only has the linkopts like what we did for nccl. Would that help your issue?